Clean-up code in block editor & fix value watcher (#19634)
directus/directus: .changeset/large-gorillas-decide.md

---
"@directus/app": patch
---

> Fixed a problem with the block editor where the "Discard all changes" function had no effect

Author: ```suggestion
Fixed an issue with the block editor where the "Discard all changes" function had no effect
```
